UefiCpuPkg/PiSmmCpuDxeSmm/CpuS3.c

```diff
@@ -243,8 +243,9 @@ InitSmmS3ResumeState (
 
     //
     // Patch SmmS3ResumeState->SmmS3Cr3
+    // The SmmS3Cr3 is only used by S3Resume PEIM to switch CPU from 32bit to 64bit
     //
-    InitSmmS3Cr3 ();
+    InitSmmS3Cr3 (&mSmmS3ResumeState->SmmS3Cr3);

```

Comment:
please use SmmS3ResumeState, not mSmmS3ResumeState

Reply:
agree